server/src/api/services/external/temple.service.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import { TEMPLE_OSRS } from '../../constants';
+import { NotFoundError } from '../../errors';
+
+/**
+ * Fetches the group members from the temple API
+ */
+async function getGroupMembers(gid: number): Promise<string[]> {
```

Comment:
Can you change to "fetchGroupMembers"? I often think of "get" as a direct data retrieval, and fetch as a remote transaction